environment/simulation/simulation.py

```diff
@@ -51,10 +51,9 @@ def __init__(self,
         # internal simulation properties
         self._simulation_is_running = True
         self._dt = 1 / 60.0
-        map_initial_x_offset = (self.initial_object_position[0] - self.map_segment_size[0])
-        self._map_beginning = (-map_initial_x_offset, 0.0)
-        self._map_middle_right_boundary = (map_initial_x_offset, 0.0)
-        self._enemy_position = -map_initial_x_offset
+        self._map_beginning = (-1.5*map_segment_size[0], 0.0)
```

Comment:
Czemu właściwie tu jest -1.5? Bo nie ogarniam trochę :D

Reply:
Może jakaś podmianka na stałą, enuma, zmienną?

Reply:
Bo zakładam, że chodzi o to, że na start na osi x widzimy początek lewego segmentu gdzieś na środku ekranu